scripts/coccinelle/reserved_function_names.cocci

```diff
@@ -0,0 +1,98 @@
+/// Find function names starting with '__' or '_' and rename them
+// Copyright: (C) 2018 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+virtual patch
+virtual report
+
+@r_anyfn disable optional_storage@
+identifier fn =~ "^_.*";
+position p;
+@@
+
+  fn@p (...)
+
+@r_staticfn disable optional_storage@
```

Comment:
I don't think `disable optional_storage` is useful here.

Reply:
Nope, fixed.